README.md

```diff
@@ -110,6 +120,19 @@ Note: when storing points, the following conventions are used:
    default, the adapter stores all numeric fields as floats. This can be changed
    by enumerating integer fields via `intFields` option.
 
+### Optimizations
```

Comment:
I have to agree with @bkutil on this one as this is a bit too much of what an actual developer of the language/adapter would care about at this point. The README should be for a user of the plugin which sticks to installing/using the adapter without going into the technical details of how the adapter works (I'm sure we've abused this elsewhere).